service/src/test/java/uk/gov/hmcts/reform/fpl/model/StandardDirectionOrderTest.java

```diff
@@ -41,4 +44,30 @@ void shouldSetDirectionsToAnEmptyList() {
 
         assertThat(standardDirectionOrder.getDirections()).isEmpty();
     }
+
```

Comment:
missing IsRemovable test

Reply:
Yeah fair enough, I did not add as it's essentially the same as `isSealed` which is tested.